zerver/openapi/zulip.yaml

```diff
@@ -4744,7 +4748,15 @@ paths:
                             the user was subscribed to as a result of the query.
                           type: array
                           items:
-                            type: string
+                            type: object
+                            description: |
+                              A dictionary containing name and ID of the stream.
```

Comment:
"containing the name and ID" -- you need to add "the" for this to be proper English.  Same with the similar one above.